validate.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
```

Comment:
A echo at the begging wouldn't hurt, like echo "====> Checking if docker build successfully" or something like that